tools/c7n_azure/c7n_azure/resources/network_security_group.py

```diff
@@ -33,56 +35,70 @@ class resource_type(object):
             'resourceGroup'
         )
 
+FROM_PORT = 'fromPort'
+TO_PORT = 'toPort'
+PORTS = 'ports'
+EXCEPT_PORTS = 'exceptPorts'
+IP_PROTOCOL = 'ipProtocol'
```

Comment:
@thisisshi re: your comment above, I put these in variables, just to maintain consistency throughout the class. The all caps and underscore is just the name of the variable, not the actual schema